tests/lib/signIn.js

```diff
@@ -60,6 +60,16 @@ define([
           );
       });
 
+      test('#with service', function () {
+        var email = "test" + new Date().getTime() + "@restmail.net";
+        var password = "iliketurtles";
+
+        return respond(client.signUp(email, password), RequestMocks.signUp)
+          .then(function (res) {
+            return respond(client.signIn(email, password, {service: 'sync'}), RequestMocks.signIn);
```

Comment:
We are not inspecting XHR requests in other tests.
We do however use sinon for certain things so we can add a spy to this test.